src/main/java/core/basesyntax/figures/Drawable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.figures;
+
+public interface Drawable {
+    default void draw() {
+        System.out.println("Unknown type of figure");
+    }

```

Comment:
you don't need to make this method default

Reply:
why?